frameworks/bitcoin-move/sources/ord.move

```diff
@@ -227,6 +226,8 @@ module bitcoin_move::ord {
                 inscription.offset = new_sat_point.offset;
 
                 // TODO handle curse inscription
+                // drop the temporary area if inscription is transferred.
+                drop_temp_area(&mut inscription_obj);
```

Comment:
We should define a custom transfer function for `Object<Inscription>`.

Reply:
It is a private function; ensure the `transfer_extend` and clean temp state are the same simultaneously.